### PR TITLE
Add a basic comment about DNA sequence simulation

### PR DESCRIPTION
**Pull request by hyanwong**
I think somewhere *right* at the top we need to say that, like other simulators, msprime can be (trivially) used to simulate DNA sequence evolution. I'm not sure my wording in this PR is quite right, so I've flagged this as a draft PR only.

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -5,7 +5,9 @@
 # Introduction
 
 This is the manual for msprime, a population genetics simulator
-based on [tskit](https://tskit.dev). Msprime can simulate
```
Comment (jeromekelleher) [inline docs/intro.md:8]: This reads strangely to me - "trivial" isn't a positive word IMO.

How about we make the first sentence "This is the manual for msprime, a population genetics simulator of ancestry and DNA sequence evolution based on tskit"

Comment (hyanwong) [reply docs/intro.md:8]: Yes, I agree about "trivial", but couldn't find the right wording. Yours sounds good.

Comment (jeromekelleher) [reply docs/intro.md:8]: OK, let's change then.

Comment (hyanwong) [reply docs/intro.md:8]: Do you want to or shall I?

Comment (jeromekelleher) [reply docs/intro.md:8]: Can you do it please

**Comment by jeromekelleher** [comment]
Can you update this please @hyanwong or close the PR?

**Comment by hyanwong** [comment]
Yep, just doing this. Do you want "msprime" to be changed to ``` {program}`msprime` ``` (and same for ``` {program}`tskit` ```) like in the rest of the docs, @jeromekelleher ?

**Comment by hyanwong** [comment]
> Yep, just doing this. Do you want "msprime" to be changed to `` {program}`msprime` `` (and same for `` {program}`tskit` ``) like in the rest of the docs, @jeromekelleher ?

I did that anyway - seemed best to be consistent.

**Review by jeromekelleher — APPROVED** [review]
LGTM. I find the {program} markup a bit tedious to be honest, but why not.

**Pull request merged**